src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
OK, you said "PluginMountPoints are shared behind the scenes.".
So let's share them directly, I really see no sense to hide this.